packages/patternfly-4/react-core/src/components/LoginPage/LoginPage.md

```diff
@@ -7,6 +7,6 @@ import LinkPreview from '@content/../LinkPreview';
 import RawSimpleLoginPage from '!!raw-loader!./examples/SimpleLoginPage';
```

Comment:
Do you need: `typescript: true` to show the TS badge?

Reply:
Yep!